docs/tutorials/intro_tutorial.rst

```diff
@@ -852,6 +853,19 @@ Or to plot the wealth of a given agent (in this example, agent 14):
 
 .. image:: intro_tutorial_files/intro_tutorial_45_1.png
 
+You can also use pandas to export the data to a CSV (comma separated value),
+which can be opened by any common spreadsheet application or opened by pandas.
+
+If you do not specify a file path, the file will be saved in the local directory.
+After you run the code below you will see two files appear (*model_data.csv* and *agent_data.csv*)
+
+.. code:: python
+
+    # save the model data (stored in the pandas gini object) to csv
```

Comment:
Was about to say `CSV`, but we have gone through too many back and forth. Later I will fix it across the repo once for all.